TypeScript.py

```diff
@@ -1,19 +1,13 @@
 import sys
+import os
 
-# Sublime/Python 2 & 3 differ in the name of this module, thus package import
-# needs to be handled slightly differently
-if sys.version_info < (3, 0):
-    from libs import *
-    from libs.reference import *
-    from libs.view_helpers import *
-    from listeners import *
-    from commands import *
-else:
-    from .libs import *
-    from .libs.reference import *
-    from .libs.view_helpers import *
-    from .listeners import *
-    from .commands import *
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
I tried this approach but it didn't work. The separate treatment at the top level will indeed make top level import successful, but the cascading import later will still fail with "No module named typescript_module". So we either end up to do the relative importing in every file, or mess with the ``sys.path``.  Related links I found: 
http://stackoverflow.com/questions/9383014/cant-import-my-own-modules-in-python
http://stackoverflow.com/questions/24722212/python-cant-find-module-in-the-same-folder